app/assets/stylesheets/application.tailwind.css

```diff
@@ -68,4 +66,22 @@
   .typography-micro-m-lh150 {
     @apply text-micro leading-150 font-medium;
   }
+  .btn-primary{
+    @apply typography-text-s-lh150 py-2 px-4 rounded bg-brand-palette-03 hover:bg-brand-palette-04 hover:no-underline text-white
+  }
+  .btn-secondary{
+    @apply typography-text-s-lh150 text-dark-gray-palette-p3 border-2 border-light-palette-p3 rounded-md px-4 py-2 hover:bg-dark-gray-palette-p3 hover:text-light-palette-p5 hover:border-transparent 
+  }
+  .btn-secondary-2{
+    @apply typography-text-s-lh150 bg-light-palette-p4 text-dark-gray-palette-p3 rounded-md px-4 py-2 hover:bg-brand-palette-07 hover:text-brand-palette-03 
+  }
+  .icon-btn-outline{
+    @apply p-2.5 border-solid border-2 border-light-palette-p3 rounded-r-lg flex-1 hover:border-brand-palette-04
+  }
+  .icon-btn-outline svg{
+    @apply hover:stroke-brand-palette-04
+  }
+  .form-input {
+    @apply py-2 px-4 bg-light-palette-p4 border-2 border-light-palette-p3 rounded-md text-dark-gray-palette-p1 typography-text-m-lh150 focus:outline-none focus:ring focus:ring-light-palette-p3 outline-none	focus:border-transparent
```

Comment:
Hmmmm aqui to achando que é `border-light-palette-p2`, vou verificar melhor

Reply:
É p3 mesmo
![image](https://github.com/douglara/woofed-crm/assets/45243355/07b7f4ae-fc2b-4405-8279-d3175882d00a)